.github/workflows/cd.yaml

```
@@ -0,0 +1,58 @@
+name: Deploy Static Website to S3
```

How about changing the name to `Deploy Website` and the name of the file to `deploy-website`?

```suggestion
name: Deploy Website
```